internal/api/utils.go

```diff
@@ -97,6 +97,8 @@ func MapQueryParameters(c echo.Context) (map[string]interface{}, error) {
 			log.Error("error parsing end_date:", err)
 			return queryParams, err
 		}
+		// Inclusive user-provided end_date timestamp
+		endTimestamp = endTimestamp.Add(24 * time.Hour)
```

Comment:
@upadhyeammit, what would prefer to have your opinion as well on this?

Reply:
It took me bit of time to understand whats happening here. I can see we are adding 24hours(a day) to user provided date for returning the recommendations upto user provided date. However the condition for filter has `<=`; so do we need it?
~~~
queryParams["recommendation_sets.monitoring_end_time <= ?"] = endTimestamp
~~~

Reply:
Removed the `=` from `queryParams["recommendation_sets.monitoring_end_time <= ?"] = endTimestamp`
Regarding `start_date` I think we should be fine, the 24 Hrs delta is being added only for user provided end_date